packages/components/tests/integration/components/hds/dropdown/list-item-test.js

```diff
@@ -1,115 +1,148 @@
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
-import { render } from '@ember/test-helpers';
+import { render, resetOnerror } from '@ember/test-helpers';
 import { hbs } from 'ember-cli-htmlbars';
 
 module('Integration | Component | hds/dropdown/list-item', function (hooks) {
   setupRenderingTest(hooks);
 
-  // ITEM: INTERACTIVE
+  hooks.afterEach(() => {
+    resetOnerror();
+  });
 
-  test('Interactive link renders', async function (assert) {
-    // Set any properties with this.set('myProperty', 'value');
-    // Handle any actions with this.set('myAction', function(val) { ... });
+  // notice (1): the "list-item" has multiple "types" controlled via the @item property
+  // notice (2): unlike other components, the `...attributes` spread is not applied to the top element, but to the `<button>/<a>` children,
+  // so we can't use the DOM "id" to target the component but we have to rely on the class name
```

Comment:
@MelSumner this is something I want to discuss with you tomorrow: I don't think we have alternatives to this, but we have to clearly document it on the web page (under Component API)